packages/flutter/lib/src/material/data_table.dart

```diff
@@ -414,14 +428,14 @@ class DataTable extends StatelessWidget {
     }
     label = Container(
       padding: padding,
-      height: _headingRowHeight,
+      height: headingRowHeight ?? _headingRowHeight,
```

Comment:
I found it confusing at first glance that there is both a `headingRowHeight` and `_headingRowHeight` (ditto for `dataRowHeight`).  The constant should probably be `_kHeadingRowHeight`, but I know you didn't write those, so probably not worth changing in this PR.

Reply:
Nevermind about the `k`, as discussed on https://github.com/flutter/flutter/pull/33628#discussion_r292634315